corehq/apps/dump_reload/tests/test_dump_models.py

```diff
@@ -195,3 +194,34 @@ def _ignore_model(model):
     ]
     assert not uncovered_models, ("Not all Django models are covered by domain dump.\n"
         + '\n'.join(sorted(uncovered_models)))
+
+
+def test_foreign_keys_for_dumped_sql_models_are_also_included():
+    IGNORE_FK_MODELS = [
+        "scheduling.SMSCallbackContent",  # Model is no longer supported, so this would only impact old domains
```

Comment:
Instead of ignoring `SMSCallbackContent`, I could add an entry in `APP_LABELS_WITH_FILTER_KWARGS_TO_DUMP` like so:
```
FilteredModelIteratorBuilder('scheduling.SMSCallbackContent', ManyFilters('alertevent__schedule__domain', 'casepropertytimedevent__schedule__domain', 'randomtimedevent__schedule__domain', 'timedevent__schedule__domain')),
```
But it didn't seem worth it since this model is likely not going to be in use anyway.

Reply:
I'm second guessing ignoring `SMSCallbackContent`. The reasoning is that it seems harder to maintain documentation/awareness around this list so future operators of the `dump_domain_data` command know to confirm the domain does not have any `SMSCallbackContent` objects than it does to just include this filter in the list of dumped models. The command already takes an obscene amount of time to run, so omitting this for that reason isn't sufficient.

TLDR; `users.Permission` is a unique exception that justifies being in `IGNORE_FK_MODELS` where `SMSCallbackContent` really isn't. What do you think @kaapstorm and @millerdev?

Reply:
Seems reasonable to include `SMSCallbackContent` if it will not add significant time to the dump.

Reply:
Yep I tested it with a few domains in production and it is very fast. I think I phrased what I attempted to say above a bit better in the commit message: 98361a36494daa3144088ec26036aca3b8647a05.